homework/style.css

```diff
@@ -1,3 +1,112 @@
 .alert-error {
-    color: red;
-}
\ No newline at end of file
+  color: red;
+}
+html {
+  display: block;
+  color: text;
+}
+body {
+  width: 768px;
+  margin-left: auto;
+  margin-right: auto;
+  background-color: #f8f8f8;
+  font-family: 'Roboto', sans-serif;
+  color: rgb(0, 0, 0, 87%);
+  margin-top: 0;
+}
+.header {
+  color: white;
+  background-color: #3f51b5;
+  padding: 8px 16px;
+  margin-bottom: 16px;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+}
+#container {
+  display: flex;
+  flex-direction: row;
+  align-items: flex-start;
+}
+.whiteframe {
+  margin-bottom: 8px;
+  border: none;
+  border-radius: 2px;
+  background-color: #fff;
+  box-shadow: 0 1px 8px 0 rgba(0, 0, 0, 0.2), 0 3px 4px 0 rgba(0, 0, 0, 0.14),
+    0 3px 3px -2px rgba(0, 0, 0, 0.12);
+}
+.leftDiv,
+.rightDiv {
+  background-color: white;
+  flex: 1;
+}
+.leftDiv {
+  padding: 16px;
+  margin-right: 16px;
+}
+table {
+  table-layout: fixed;
+  color: rgb(0, 0, 0, 81%);
+}
+.contributor-header {
+  font-size: 0.8rem;
+  color: rgb(0, 0, 0, 54%);
+  padding: 8px 16px 4px 16px;
+}
+.contributor-list {
+  list-style-type: none;
+  padding: 0;
+  margin: 0;
+}
+.contributor-item {
+  border-bottom: solid 1px rgb(0, 0, 0, 12%);
+  padding: 16px;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  cursor: pointer;
```

Comment:
To get rid of the underlines in the list items you can add:

```css
text-decoration: none;
```

And you can remove `cursor: pointer` as that is the default for an anchor tag.

Reply:
yes. I applied text-decoration, removed th cursor thing.